.github/workflows/dependabot.yml

```diff
@@ -37,11 +39,19 @@ jobs:
 
         # deduplicate lockfile
         yarn deduplicate
+      env:
+        YARN_ENABLE_SCRIPTS: 0 # disable postinstall scripts
+    - name: Config Git
+      run: |
+        # use personal access token to allow triggering new workflow
```

Comment:
Should this be `to avoid triggering a new workflow`?

Reply:
@kevinbarabash No, that is the exact purpose. Currently, autofix commits don't run the tests automatically.